hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/pipeline/PipelineFactory.java

```diff
@@ -35,7 +35,7 @@
 /**
  * Creates pipeline based on replication type.
  */
-public final class PipelineFactory {
+public class PipelineFactory {
```

Comment:
The access can be made package-private?

Reply:
This is extended in org.apache.hadoop.ozone.recon.scm.ReconPipelineFactory.